.github/workflows/ci.yml

```diff
@@ -180,7 +180,7 @@ jobs:
           if [[ -n ${MODULES_ARG} ]]; then
             echo "Running modules: ${MODULES_ARG}"
             mvn -fae -V -B -s .github/mvn-settings.xml -fae -Dall-modules \
-                        -Dinclude.quarkus-cli-tests -Dts.quarkus.cli.cmd="${PWD}/quarkus-dev-cli" \
+                        -Dts.quarkus.cli.cmd="${PWD}/quarkus-dev-cli" \
```

Comment:
So, if I understand it right, this is not strictly necessary, because the `quarkus.build.skip` in the `quarkus-cli` module should be enough to not do native build on that module.

But we do it anyway, because:
1.  Why enable CLI tests in native pipeline, when the CLI module is not enabled for native build anyway.
2. CLI tests in general do not make much sense in native mode.

Is that correct, or did I miss anything?

If so, than this change (and the second one) is ok.

Reply:
You understand it right, and point one is correct. 
I presume, that there may be cli tests in our ts, which a) make sense in native mode b) are not called `QuarkusCli*IT.java`(so they are not affected by `include.quarkus-cli-tests`). This is why I left `ts.quarkus.cli.cmd` as is.
There are no such tests in quarkus-cli module, though.

A counter argument against this change, is that in (unlikely?) case, if we decide to add some native-related tests into quakus-cli, we would have to remember to fix ci scripts as well.

Reply:
In that case I prefer to discard this change.
It is IMO more systematic to not exclude all CLI test in the pipeline, but rather exclude individual modules or tests.

Reply:
The change discarded